apps/webpack.js

```diff
@@ -18,6 +18,9 @@ var baseConfig = {
       '@cdo/weblab/locale': path.resolve(__dirname, 'src', 'weblab', 'locale-do-not-import.js'),
       '@cdo/apps': path.resolve(__dirname, 'src'),
       repl: path.resolve(__dirname, 'src/noop'),
+      'jquery': path.resolve(__dirname, 'node_modules', 'jquery', 'dist', 'jquery.js'),
```

Comment:
some of the new dependencies pull in their own version of jquery, resulting in two different jqueries being loaded on the page. This forces them all to use the one at the top level.